build/fuses/build.py

```diff
@@ -50,7 +51,7 @@
 """
 
 with open(os.path.join(dir_path, "fuses.json5"), 'r') as f:
-  fuse_defaults = json.loads(''.join(line for line in f.readlines() if not line.strip()[0] == "/"))
+  fuse_defaults = json.loads(''.join(line for line in f.readlines() if not line.strip()[0] == "/"), object_pairs_hook=OrderedDict)
```

Comment:
This will sort the keys alphabetically, right? Is that what we want?

Reply:
No, this will keep keys in the order in which they are added.

> dict subclass that remembers the order entries were added

And the JSON loader adds them in order of appearance.